vsphere/datadog_checks/vsphere/config.py

```diff
@@ -126,6 +126,32 @@ def __init__(self, instance, init_config, log):
             instance.get("event_resource_filters", DEFAULT_EVENT_RESOURCES)
         )
         self.include_events = instance.get("include_events", None)
+        if self.include_events is None:
+            self.exclude_filters = {
```

Comment:
We are still defining this twice/hardcoding. Could we potentially define it once in constants.py?

Reply:
Makes sense, I only define the hardcoded value once in constants.py.

Reply:
Can we use it here too?

Reply:
oops haha removed.